modules/kmm-hub-hub-and-spoke.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-hub-hub-and-spoke_{context}"]
+= KMM hub & spoke
+
+In Kernel Module Management (KMM) hub & spoke scenarios, many spoke clusters are connected to a central, powerful hub cluster. Open Cluster Management (OCM) is the reference open source implementation of this pattern, and KMM depends on it to operate in those scenarios.
```

Comment:
Red Hat documentation should point to RHACM ( Red Hat Advanced Cluster Management and not to the Upstream counterpart. 
Even the Midstream document points to that: https://openshift-kmm.netlify.app/documentation/hub_spoke/

Reply:
Rewrote these sections to refer to RHACM M/S content.